### DISCUSSION TITLE
Simpler Installation with HACS installed.

### DISCUSSION THREAD

**Discussion by trooperthorn** [sbidy/wiz_light#149] [Ideas]
I noticed the documentation walks through SSH. I found any user should be able to add using the WebUI only if the user already has HACS installed. I have Home Assistant installed with Docker and wanted to do install this without calling the containers cli.

**Install and Setup using HACS**

1. Logon to your HA or HASS WebUI
2. Click on HACS> Integrations
![wiz](https://user-images.githubusercontent.com/8163700/121420524-cd02e980-c932-11eb-82b3-1477b9d195d9.png)
3. Select the 3 vertical ... at the top Right.
4. Select Custom Repositories, then enter the following:
      Custom Repository URL: https://github.com/sbidy/wiz_light
      Category: Integration
5. Select Add
6. Reboot to Apply.
7. Configuration> Integrations> Add Integration, then search WiZ
8. Enter IP Address and Name to give the Light, then select Add.

**Comment by stefsmeets** [comment]
Thanks for this, works great! Had to update from an old version of `wiz_light` now that HA requires the version to be set in the manifest, and this made it really straightforward. 🚀
